<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ObjectiveDialog</class>
 <widget class="QDialog" name="ObjectiveDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>386</width>
    <height>282</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>240</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="objectiveFunctionGroup">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>341</width>
     <height>211</height>
    </rect>
   </property>
   <property name="title">
    <string>Fitness Functions</string>
   </property>
   <widget class="QRadioButton" name="liftDrag">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>221</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Lift Drag Ratio</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">objective</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="maxLift">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>221</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Max Lift</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">objective</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="minDrag">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>221</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Min Drag</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">objective</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="maxDownforce">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>221</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Max Downforce</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">objective</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="minLift">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>221</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Min Lift</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">objective</string>
    </attribute>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ObjectiveDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ObjectiveDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="objective"/>
 </buttongroups>
</ui>
